commerce/endpoint/account/address/classes/UsZipCleansingSample.cls

```diff
@@ -0,0 +1,32 @@
+public with sharing class UsZipCleansingSample {
```

Comment:
Why with sharing?

Reply:
@dbarbato-sf I thought we always wanted to use that by default or it would be scrutinized as a potential security issue. Can you help me understand why that is not the case?

Reply:
I should have been more verbose :) As this is sample code that we're using to educate, we should note why we're using with sharing, not without sharing in the comment.